src/main/java/frc/lib/team6328/util/FieldConstants.java

```diff
@@ -11,41 +11,96 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.math.util.Units;
-
-@java.lang.SuppressWarnings({"java:S1118", "java:S115", "java:S2386"})
+import static edu.wpi.first.apriltag.AprilTagFields.k2024Crescendo;
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.math.util.Units;
+import java.io.IOException;
 
 /**
  * Contains various field dimensions and useful reference points. Dimensions are in meters, and sets
- * of corners start in the lower left moving clockwise.
+ * of corners start in the lower left moving clockwise. <b>All units in Meters</b> <br>
+ * <br>
  *
  * <p>All translations and poses are stored with the origin at the rightmost point on the BLUE
- * ALLIANCE wall. Use the {@link #flipForRedSide(Translation2d)} and {@link #flipForRedSide(Pose2d)}
- * methods to flip these values to get the red alliance version.
+ * ALLIANCE wall.<br>
+ * <br>
+ * Length refers to the <i>x</i> direction (as described by wpilib) <br>
+ * Width refers to the <i>y</i> direction (as described by wpilib)
  */
-public final class FieldConstants {
-  public static final double fieldLength = Units.inchesToMeters(651.25);
-  public static final double fieldWidth = Units.inchesToMeters(323.25);
-  public static final double tapeWidth = Units.inchesToMeters(2.0);
-
-  /**
-   * Flips a translation for a point on the blue side of the field to the corresponding point on the
-   * red side. By default, all translations and poses in {@link FieldConstants} are stored with the
-   * origin at the rightmost point on the BLUE ALLIANCE wall.
-   */
-  public static Translation2d flipForRedSide(Translation2d translation) {
```

Comment:
This year, we only have one origin on our field (the corner to the right of the blue driverstation), so we will need to make constants for all the elements on the red side of the field as well. This method will make creating the red field constants much easier.